pkg/manager/member/tidb_member_manager.go

```diff
@@ -14,20 +14,25 @@
 package member
 
 import (
+	"context"
 	"crypto/tls"
+	"database/sql"
 	"fmt"
 	"path"
 	"strconv"
 	"strings"
+	"time"
 
+	"k8s.io/apimachinery/pkg/util/wait"
```

Comment:
Usually three parts for the import files:

- system imports
- imports in the same repo
- imports of the 3rd parties
So please move this to block L36.